documentation/book/ref-minimum-resource-requirements.adoc

```diff
@@ -0,0 +1,16 @@
+[id='minimum-resource-requirements-{context}']
+= Minimum Resource Requirements
+
+[id='cluster-operator-resource-requirements-{context}']
+== Cluster Operator resource requirements
+
+Testing has shown that the Cluster Operator functions adequately with 256Mi of memory and 200m CPU when watching two clusters.
+It is therefore recommended to use these as a minimum when configuring resource requests and not to run it with lower limits than these.
+Configuring more generous limits is recommended, especially when it is controlling multiple clusters.
+
+[id='topic-operator-resource-requirements-{context}']
+== Topic Operator resource requirements
+
+Testing has shown that the topic operator functions adequately with 96Mi of memory and 100m CPU when watching two topics.
+It is therefore recommended to use these as a minimum when configuring resource requests and not to run it with lower limits than these. 
+If the Kafka cluster has more than a handful of topics more generous requests and limits will be necessary.
```

Comment:
handful is very generic, can we be more specific? 5, 10, 15?

Reply:
Not currently, and anyway as requested by @scholzj this reference isn't actually included anywhere at the moment, since this information is of minimal real world use. I could just delete this reference if you prefer.

Reply:
We can leave it for now, we'll probably come up with a way to improve on it later.